spec.html

```diff
@@ -4259,7 +4259,7 @@
         1. Let _keys_ be ? _O_.[[OwnPropertyKeys]]().
         1. If _level_ is `"sealed"`, then
           1. Repeat for each element _k_ of _keys_,
-            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{ [[Configurable]]: *false*}).
+            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{[[Configurable]]: *false*}).
```

Comment:
Looking at all record constructors and set-of-attribute constructors, I get 108 with a space after the left brace, and 120 without. Personally, I agree it's more readable with the space.

(As for the space *before* the left brace, I get 164 with, 64 without.)